### DISCUSSION TITLE
How to load up these m3u files?

### DISCUSSION THREAD

**Discussion by TheMikie** [iptv-org/community#181] [Q&A] [answered]
Hey,   which app do I need to use to load these m3u files.   My  SuperTV in Brasil recently took the dump and I lightly explored Kodi and well yeah that definitely seems dead.  Came across the Brazilian IPTV repository but looking for some assistance.   

Can anyone please point me in the right direction how I can load up this channel list in what app?

**Comment by what-a** [comment, marked as answer]
https://github.com/iptv-org/awesome-iptv#apps
